red/runtime/util.js

```diff
@@ -328,6 +328,11 @@ function evaluateNodeProperty(value, type, node, msg) {
     return value;
 }
 
+function normaliseRegisterTypeName(name) {
```

Comment:
Thank you very much for pointing it out.  I fixed both of them and push the code.